pages/apis/graphql/graphql_cookbook.md

```diff
@@ -596,6 +596,114 @@ mutation {
 }
 ```
 
+## Clusters
+
+A collection of common tasks with clusters using the GraphQL API.
+
+### List cluster IDs
+
+Get the first 10 clusters and their information for a particular organization:
```

Comment:
Style: Just reducing the use of "particular" through this section because I don't think it's necessary here.

```suggestion
Get the first 10 clusters and their information for an organization:
```

Reply:
TIA @mbelton-buildkite - amended 👍